examples/scalajs-react/index-dev.html

```diff
@@ -0,0 +1,30 @@
+<!doctype html>
+<html lang="en" data-framework="scalajs-react">
+<head>
+    <meta charset="utf-8">
+    <title>scalajs-react (dev) • TodoMVC</title>
+    <link rel="stylesheet" href="node_modules/todomvc-common/base.css">
+    <link rel="stylesheet" href="node_modules/todomvc-app-css/index.css">
+</head>
+<body>
+<section id="todoapp" class="todoapp"></section>
+<footer class="info">
+    <p>Double-click to edit a todo</p>
+
+    <p>Created by <a href="http://github.com/elacin/">elacin</a></p>
+
+    <p>Part of <a href="http://todomvc.com">TodoMVC</a></p>
+</footer>
+
+<script src="node_modules/todomvc-common/base.js"></script>
+<script type="text/javascript" src="generated/todomvc-jsdeps.js"></script>
+<script type="text/javascript" src="target/scala-2.11/todomvc-fastopt.js"></script>
+
+<script type="text/javascript">
```

Comment:
The same could be said for the entire index page, It's there just for bootstrapping purposes - the point of this whole ordeal was exactly to avoid writing this stuff myself, hehe.

Anyways, I'll be removing that type, then